src/subsets.c

```diff
@@ -39,11 +39,9 @@
 #include "unicode.h"
 #include "unicode_range.h"
 
-#define SUBSET_DEBUG 1
-
```

Comment:
As you can see, this was always true but I have yet to see an overflow in `powi()` or `fac()` with normal usage (or actually in any usage) of this mode. I decided to drop the macros and continue always doing the checks.